tests/tape/tapes/test_qnode.py

```diff
@@ -20,6 +20,10 @@
 from pennylane.tape import JacobianTape, QNode, draw, qnode, QubitParamShiftTape, CVParamShiftTape
 
 
+def dummyfunc():
+    return None

```

Comment:
Unless we want to add an extra line that asks if the qfunc is None in the qnode, we need to be more careful with the tests here

Reply:
This makes sense. The fact that `func` was never validated was something I took advantage off when test making, that I won't be able to any more :(

(side note: it's purely stylistic, but you could also do `qml.QNode(lambda: None, ...)`._